### PR TITLE
[JENKINS-66639] adding explict jcasc symbol on CertificateCredentialImpl and UsernamePasswordCredentialsImpl classes

### PR DESCRIPTION
**Pull request by aHenryJard**
https://issues.jenkins.io/browse/JENKINS-66639

For this specific feature, I have run test before and after adding the Symbol: both run are fine.

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Comment by Vlatombe** [comment]
Unrelated changes in `SystemCredentialsTest` ?

**Review by jglick — APPROVED** [review]
Other than the missing test case in `SystemCredentialsTest` this looks reasonable.

Do not forget about the other most commonly used implementations, in `ssh-credentials` and `plain-credentials`, maybe also `aws-credentials`.

File: src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java [hunk 1]
```diff
@@ -63,4 +78,41 @@ public void setConfig(String config) {
...
+
+    @Test
+    public void exportCertificateCredentialsImplConfiguration() throws Exception {
+        CertificateCredentialsImpl usernamePasswordCredentials =
+                new CertificateCredentialsImpl(CredentialsScope.GLOBAL,
+                                               "credential-certificate",
+                                               "Credential with certificate",
+                                               "password",
+                                               new CertificateCredentialsImpl.UploadedKeyStoreSource(Base64.encode(
+                                                       "Testing not real certificate".getBytes())));
```
Comment (Vlatombe) [inline src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java:108]: Use `java.util.Base64` instead.
```suggestion
                                               new CertificateCredentialsImpl.UploadedKeyStoreSource(Base64.getEncoder().encode("Testing not real certificate".getBytes())));
```

File: src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+import com.cloudbees.plugins.credentials.domains.Domain;
+import com.cloudbees.plugins.credentials.impl.CertificateCredentialsImpl;
+import com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl;
+
 import hudson.Extension;
 import hudson.ExtensionList;
 import io.jenkins.plugins.casc.ConfigurationAsCode;
 import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
 import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import jcifs.util.Base64;
```
Comment (jglick) [inline src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java:17]: Use the Java Platform version.

Comment (Vlatombe) [reply src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java:17]: The trap of merging GitHub suggested changes 😄

Comment (Wadeck) [reply src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java:17]: Or just `SecretBytes.fromString` ;)

**Comment by Wadeck** [comment]
(Will merge once CI passes)

**Pull request merged**